assets/css/planning.scss

```diff
@@ -169,3 +169,15 @@ $tableBoxSize: 40px;
     }
   }
 }
+
+@media screen and (min-width: 1001px) {
```

Comment:
Those values should be the default one, and you should only use media query for the mobile versions